CheckDrive.Web/CheckDrive.Web/Views/Shared/_CarDetailsForMechanicHandover.cshtml

```diff
@@ -2,61 +2,111 @@
+    <div class="grid-container mt-3">
+        <div class="row content">
+            <div class="col text-center">
+                Model
+            </div>
+            <div class="col text-center">
+                Rangi
+            </div>
+            <div class="col text-center">
+                Davlat raqami
+            </div>
+        </div>
+        <div class="row value">
+            <div class="col text-center">
+                @Model.Model
+            </div>
+            <div class="col text-center">
+                @Model.Color
+            </div>
+            <div class="col text-center">
+                @Model.Number
+            </div>
+        </div>
+        <div class="row content">
+            <div class="col text-center">
+                Ishlab chiqarilgan yil
+            </div>
+            <div class="col text-center">
+                Bosib o'tgan masofasi
+            </div>
+            <div class="col text-center">
+                Yoqilg'i qoldig'i
+            </div>
+        </div>
+        <div class="row value">
+            <div class="col text-center">
+                @Model.ManufacturedYear
+            </div>
+            <div class="col text-center">
+                @Model.Mileage
+            </div>
+            <div class="col text-center">
+                @Model.RemainingFuel
+            </div>
+        </div>
+        <div class="row content">
+            <div class="col text-center">
+                O'rtacha yoqilg'i sarfi
+            </div>
+            <div class="col text-center">
+                Yoqilg'i baki sig'imi
+            </div>
+        </div>
+        <div class="row value">
+            <div class="col text-center">
+                @Model.MeduimFuelConsumption
+            </div>
+            <div class="col text-center">
+                @Model.FuelTankCapacity
+            </div>
+        </div>
+        <div class="row content">
+            <div class="col text-center">
+                1 yillik bosib o'tishi me'yori
+            </div>
+            <div class="col text-center">
+                1 oylik bosib o'tish me'yori
+            </div>
+        </div>
+        <div class="row value">
+            <div class="col text-center">
+                @Model.OneYearMediumDistance
+            </div>
+            <div class="col text-center">
+                @Model.OneMonthMediumDistance
+            </div>
+        </div>
+        <div class="row content">
+            <div class="col text-center">
+                1 yillik yoqilg'i me'yori
+            </div>
+            <div class="col text-center">
+                1 oylik yoqilg'i me'yori
+            </div>
+        </div>
+        <div class="row value">
+            <div class="col text-center" style="border-bottom: 0.1px solid #d9d9d9;">
+                @Model.OneYearMeduimFuelConsumption
+            </div>
+            <div class="col text-center" style="border-bottom: 0.1px solid #d9d9d9;">
```

Comment:
There why you use inline css ,  You can replace css to site.css file or create css file in root folder .

Reply:
I agree with you